edgeless_api/src/function_instance/mod.rs

```diff
@@ -48,6 +67,9 @@ pub trait FunctionInstanceAPI: FunctionInstanceAPIClone + Sync + Send {
     async fn start(&mut self, spawn_request: SpawnFunctionRequest) -> anyhow::Result<SpawnFunctionResponse>;
     async fn stop(&mut self, id: InstanceId) -> anyhow::Result<()>;
     async fn update_links(&mut self, update: UpdateFunctionLinksRequest) -> anyhow::Result<()>;
+    async fn update_node(&mut self, request: UpdateNodeRequest) -> anyhow::Result<UpdateNodeResponse>;
```

Comment:
It depends on whether we would like to have one API (s07) for both the orchestrator and the nodes or two APIs (s07-orc vs. s07-node, or s07 vs. sXX new number).

For now having just one was quicker so I didn't even spend time thinking on what is best. We can discuss a refactoring anytime (this will be part of the activities of WP2, too).